### PR TITLE
update market insight

### PR DESCRIPTION
**Pull request by minjikarin**
update market insight
remove the old categories

### CODE REVIEW

**Comment by minjikarin** [comment]
@nicoongit, this PR is a part of iterations to go for 100% completion of docs (info in confluence), can you please review?

File: docs/resources/market-insight.md [hunk 1]
```diff
@@ -1 +1,247 @@
...
+
+### The tops destinations by booking volume 
+
+`Flight Most Booked Destinations API` returns the most booked destinations from a given city. 
+
+```bash
+GET https://test.api.amadeus.com/v1/travel/analytics/air-traffic/booked?originCityCode=NCE&period=2018-01
+```
+
+### the busiest month/period by air traffic
```
Comment (nicoongit) [inline docs/resources/market-insight.md:37]:
```suggestion
### The busiest month/period by air traffic 
```

File: docs/resources/market-insight.md [hunk 2]
```diff
@@ -1 +1,247 @@
...
+
+`Flight Busiest Traveling Period API` returns the peak periods for travel to/from a specific city. 
+
+```bash
+GET https://test.api.amadeus.com/v1/travel/analytics/air-traffic/busiest-period?cityCode=NCE&period=2018
+```
+
+### Response 
+
+The 3 endpoints return in same structure.
```
Comment (nicoongit) [inline docs/resources/market-insight.md:47]:
```suggestion
The three endpoints have the same response structure.
```

File: docs/resources/market-insight.md [hunk 3]
```diff
@@ -1 +1,247 @@
...
+                "travelers": {
+                    "score": 33
+                }
+            }
+        }
+    ]
+}
+```
+
+- `subType` is `BOOKED` or `TRAVELED` depending on the endpoint.
```
Comment (nicoongit) [inline docs/resources/market-insight.md:132]:
```suggestion
- `subType` is `BOOKED` or `TRAVELED`, depending on the endpoint. 
```

File: docs/resources/market-insight.md [hunk 4]
```diff
@@ -1 +1,247 @@
...
+
+### The top destinations by passenger volume 
+
+`Flight Most Traveled Destinations API` returns the most visited destinations from a given city. 
+
+```bash
+GET https://test.api.amadeus.com/v1/travel/analytics/air-traffic/traveled?originCityCode=NCE&period=2018-01
+```
+
+### The tops destinations by booking volume
```
Comment (nicoongit) [inline docs/resources/market-insight.md:29]:
```suggestion
### The top destinations by booking volume 
```

File: docs/resources/market-insight.md [hunk 5]
```diff
@@ -1 +1,247 @@
...
+
+
+### Sorting
+
+Sorting is possible for the top destinations' endpoints. 
+
+- analytics.flights.score - sort destination by flights score (decreasing)
+- analytics.travelers.score - sort destination by traveler's score (decreasing)
+
+for example :
```
Comment (nicoongit) [inline docs/resources/market-insight.md:143]:
```suggestion
For example : 
```

File: docs/resources/market-insight.md [hunk 6]
```diff
@@ -1 +1,247 @@
...
+}
+```
+
+- `subType` is `BOOKED` or `TRAVELED` depending on the endpoint. 
+- In `analytics`, the `score` in `flight` is flights to this destination as a percentage of total departures, and the `score` in `traveler` is the number of passengers traveling to the destination as a percentage of total passenger departures.
+
+
+### Sorting
+
+Sorting is possible for the top destinations' endpoints.
```
Comment (nicoongit) [inline docs/resources/market-insight.md:138]:
```suggestion
Sorting is enabled on the "top destinations" endpoints. 
```

File: docs/resources/market-insight.md [hunk 7]
```diff
@@ -1 +1,247 @@
...
+
+for example : 
+
+```bash
+GET https://test.api.amadeus.com/v1/travel/analytics/air-traffic/traveled?originCityCode=NCE&period=2018-01&sort=analytics.travelers.score
+```
+
+### Direction 
+
+For the busiest period of travel insight from a given city, you can specify the direction either `ARRIVING` to have statistics on travelers or `DEPARTING` for statistics on travelers leaving the city. By default, statistics are given on travelers ARRIVING the city.
```
Comment (nicoongit) [inline docs/resources/market-insight.md:151]:
```suggestion
For the Flight Busiest Traveling Period API, you can specify the direction as:

* `ARRIVING` for statistics on travelers arriving in the city
* `DEPARTING` for statistics on travelers leaving the city

By default, statistics are given on travelers ARRIVING in the city.
```

File: docs/resources/market-insight.md [hunk 8]
```diff
@@ -1 +1,247 @@
...
+Apart from the top destinations and busiest period insight in a city, you can also help users gain insights into a neighborhood, hotel, or vacation rental with the `Location Scores API`. 
+
+For a given latitude and longitude, it provides popularity scores for the following leisure and tourism categories:
+
+- Sightseeing
+- Restaurants
+- Shopping
+- Nightlife
+
+For each category, the API provides an overall popularity score and scores for select subcategories
```
Comment (nicoongit) [inline docs/resources/market-insight.md:168]:
```suggestion
For each category, the API provides an overall popularity score as well as scores for select subcategories 
```

File: docs/resources/market-insight.md [hunk 9]
```diff
@@ -1 +1,247 @@
...
+
+For a given latitude and longitude, it provides popularity scores for the following leisure and tourism categories:
+
+- Sightseeing
+- Restaurants
+- Shopping
+- Nightlife
+
+For each category, the API provides an overall popularity score and scores for select subcategories 
+like luxury shopping, vegetarian restaurants, or historical sights, among others. Location scores are on
```
Comment (nicoongit) [inline docs/resources/market-insight.md:169]:
```suggestion
such as luxury shopping, vegetarian restaurants, or historical sights. Location scores are on 
```

File: docs/resources/market-insight.md [hunk 10]
```diff
@@ -1 +1,247 @@
...
+For a given latitude and longitude, it provides popularity scores for the following leisure and tourism categories:
+
+- Sightseeing
+- Restaurants
+- Shopping
+- Nightlife
+
+For each category, the API provides an overall popularity score and scores for select subcategories 
+like luxury shopping, vegetarian restaurants, or historical sights, among others. Location scores are on 
+a simple 0-100 scale and are powered by the [AVUXI TopPlace](https://www.avuxi.com/topplace/location-scores) algorithm which analyzes millions of online reviews, comments, and points of interest.
```
Comment (nicoongit) [inline docs/resources/market-insight.md:170]:
```suggestion
a simple 0-100 scale and are powered by the [AVUXI TopPlace](https://www.avuxi.com/topplace/location-scores) algorithm, which analyzes millions of online reviews, comments, and points of interest.
```

File: docs/resources/market-insight.md [hunk 11]
```diff
@@ -1 +1,247 @@
-to be updated
+# Market insights
+
+With Amadeus Self-Service APIs, you can get insights from millions of bookings and our technology partners.  In the `Market insights` category, we have 4 APIs available.
```
Comment (nicoongit) [inline docs/resources/market-insight.md:3]:
```suggestion
With Amadeus Self-Service APIs, you can get insights from millions of bookings and our technology partners.  In the `Market insights` category, we have four APIs available.
```

File: docs/resources/market-insight.md [hunk 12]
```diff
@@ -1 +1,247 @@
...
+| [ Flight Most Booked Destinations API](https://developers.amadeus.com/self-service/category/air/api-doc/flight-most-booked-destinations/api-reference )           | See the top destinations by booking volume for a given city and month.                  |
+| [Flight Busiest Traveling Period  API](https://developers.amadeus.com/self-service/category/air/api-doc/flight-busiest-traveling-period/api-reference)        | See monthly air traffic levels by city to understand season trends.                     |
+| [Location Score API](https://developers.amadeus.com/self-service/category/destination-content/api-doc/location-score/api-reference)                               | Assess a neighborhood’s popularity for sightseeing, shopping, eating out, or nightlife. |
+
+
+## Find the top destinations or the busiest period for a given city
+
+You may wonder which destination the travelers travel to the most and when is the busiest period from a given city. You can get the travel insight from a given city and month with 3 endpoints.
```
Comment (nicoongit) [inline docs/resources/market-insight.md:15]:
```suggestion
You may wonder which destination the travelers travel to the most and when is the busiest period for a given city. You can get the travel insight from a given city and month with three endpoints.
```

File: docs/resources/market-insight.md [hunk 13]
```diff
@@ -1 +1,247 @@
...
+
+!!! information
+    - The results of these 3 endpoints are based on estimated flight traffic summary data from the past 12 months.
```
Comment (nicoongit) [inline docs/resources/market-insight.md:18]:
```suggestion
    - The results of these three endpoints are based on estimated flight traffic summary data from the past 12 months. 
```

**Comment by minjikarin** [comment]
Thank you so much @nicoongit !!!!

**Pull request merged**